package.json

```diff
@@ -6,30 +6,20 @@
   "type": "module",
   "author": "Netlify Inc.",
   "scripts": {
-    "test": "run-s format test:dev",
-    "format": "run-s format:check-fix:*",
-    "format:ci": "run-s format:check:*",
-    "format:check-fix:lint": "run-e format:check:lint format:fix:lint",
-    "format:check:lint": "cross-env-shell eslint $npm_package_config_eslint",
-    "format:fix:lint": "cross-env-shell eslint --fix $npm_package_config_eslint",
-    "format:check-fix:prettier": "run-e format:check:prettier format:fix:prettier",
-    "format:check:prettier": "cross-env-shell prettier --check $npm_package_config_prettier",
-    "format:fix:prettier": "cross-env-shell prettier --write $npm_package_config_prettier",
-    "format:check-fix:tsd": "run-e format:check:tsd format:fix:tsd",
-    "format:check:tsd": "tsd packages/build",
-    "test:dev": "npx lerna run test",
-    "test:ci": "npx lerna run test:ci",
-    "prepare": "husky install node_modules/@netlify/eslint-config-node/.husky/",
+    "prepare": "husky install",
+    "lint": "eslint --cache --ignore-path .gitignore --max-warnings=0 . --fix",
+    "lint:ci": "eslint --cache --ignore-path .gitignore --max-warnings=0 .",
```

Comment:
should we add [–report-unused-disable-directives](https://eslint.org/docs/latest/user-guide/command-line-interface#--report-unused-disable-directives)

Reply:
Good idea I thought this is done by default 🤔 But yea!

Reply:
added the flag and it cleaned up a lot of ignores! 🥳